### ISSUE TITLE
ThreadInterruptedException is not handled correctly in task

### ISSUE THREAD

**Issue by alien11689** [myzhan/locust4j#21]
When I set step load parameters on master (e.g. step users 100 and duration 60s), than master sends new hatch messsage with 60s interval, but old threads are not stopped. 

It's because Abstract task executor swallows InterruptedException which is thrown by my task in `Thread.sleep`:

```
2020-07-31 12:38:31,514 [locust4j-worker#13] ERROR c.g.myzhan.locust4j.AbstractTask - Unknown exception when executing the task
java.lang.InterruptedException: sleep interrupted
```

and task thread cannot be stopped

To fix it I have to throw `Error` instead of `InterruptedException` what stops the task, but looks ugly.

**Comment by myzhan** [comment]
Can you show me the code?

**Comment by myzhan** [comment]
Thanks @alien11689 , it's a bug when working with --step-load. The master repeatly send hatch message instead of stopping previous tasks.

**Comment by myzhan** [comment]
See also https://github.com/myzhan/locust4j/commit/3004ae92e4fd2329f8ddf2286fc2e45458aa86c9

**Comment by myzhan** [comment]
Fixed in version 1.0.11
